source/Target/CMakeLists.txt

```diff
@@ -1,5 +1,13 @@
 include_directories(../Plugins/Process/Utility)
 
+# libswiftReflection is placed in a subdirectory which depends
+# on the host we're currently running on.
+# FIXME: There must be a better way to get at libswiftReflection.a
+string(REPLACE "-" ";" TRIPLE_LIST "${LLVM_HOST_TRIPLE}")
```

Comment:
The error message in the failing bot seems to indicate that `TRIPLE_LIST` is empty when the next line (`list(GET...)`) is called, but only on Linux.

Reply:
The issue is more complicated than that.
Linux doesn't really include the arch when building.

```
davide@rad:~/work/build$ find . -name "libswiftReflection*"
./Ninja-ReleaseAssert/swift-linux-x86_64/lib/swift/linux/x86_64/libswiftReflection.a
./Ninja-ReleaseAssert/swift-linux-x86_64/lib/swift/linux/libswiftReflection.a
```

Even though I change `CmakeLists.txt` to just link `swiftReflection`, the library is not found, as it's not exported correctly.